zerver/lib/types.py

```diff
@@ -7,7 +7,7 @@
 # including many examples
 Validator = Callable[[str, object], Optional[str]]
 ExtendedValidator = Callable[[str, str, object], Optional[str]]
-RealmUserValidator = Callable[[int, object, bool], Optional[str]]
+RealmUserValidator = Callable[[int, List[int], bool], Optional[str]]
```

Comment:
Could you clarify the reason for this change? You are moving away from case ~`Any` (matching/accepting all types), to just accepting `List[int]`, so the error you see from mypy might be varied if you define this as the `Union` you use elsewhere? You might consider extracting this type as a definition somewhere, given it is used in so many places.